collect_app/src/main/java/org/odk/collect/android/adapters/AbstractSelectListAdapter.java

```diff
@@ -227,11 +227,22 @@ public void addMediaFromChoice(AudioVideoImageTextLabel audioVideoImageTextLabel
             String imageURI = getImageURI(index, items);
             String videoURI = props.getPrompt().getSpecialFormSelectChoiceText(item, "video");
             String bigImageURI = props.getPrompt().getSpecialFormSelectChoiceText(item, "big-image");
-            audioVideoImageTextLabel.setImageVideo(imageURI, videoURI, bigImageURI, props.getReferenceManager());
-
             String audioURI = getPlayableAudioURI(props.getPrompt(), item, props.getReferenceManager());
-            if (audioURI != null) {
-                audioVideoImageTextLabel.setAudio(audioURI, props.getAudioHelper());
+            try {
+                if (imageURI != null) {
```

Comment:
I need to have those checks here not in setters because I want to pass files to those setters not URIs so that I can mock such files for tests (if a file exist or not).